typescript/phoenix-sdk/examples/seatManager.ts

```diff
@@ -0,0 +1,140 @@
+import { Connection, PublicKey, Transaction, Keypair } from "@solana/web3.js";
+import {
+  getSeatManagerAddress,
+  deserializeSeatManagerData,
+  getClaimSeatIx,
+  getEvictSeatIx,
+} from "../src/utils/seatManager";
+import * as assert from "assert";
+import base58 from "bs58";
+
+import * as Phoenix from "../src";
+import { createClaimSeatInstruction } from "../psm-src";
+
+// Ex: ts-node tests/seatManager.ts
+export async function getSeatManager() {
+  let endpoint = "https://api.devnet.solana.com";
+  const connection = new Connection(endpoint);
+  const phoenix = await Phoenix.Client.create(connection, endpoint);
+  let marketAddress = new PublicKey(
+    "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
+  );
+
+  let seatManagerAddress = getSeatManagerAddress(marketAddress);
+  let seatManagerAccountINfo = await phoenix.connection.getAccountInfo(
+    seatManagerAddress
+  );
+  if (!seatManagerAccountINfo)
+    throw new Error(
+      "Seat Manager Account not found for market: " + marketAddress?.toBase58()
+    );
+  const buffer = Buffer.from(seatManagerAccountINfo.data);
+  let seatManagerObject = deserializeSeatManagerData(buffer);
+  let seatManagerAuthority = new PublicKey(
+    "AJbjEM3LCmFZq57fKg9Cmz4RaXyZn48H6T5KBwRsvF81"
+  );
+
+  assert.equal(
+    seatManagerObject.authority.toBase58(),
+    seatManagerAuthority.toBase58()
+  );
+  assert.equal(seatManagerObject.market.toBase58(), marketAddress.toBase58());
+  console.log("Seat Manager Market: ", seatManagerObject.market.toBase58());
+  console.log(
+    "Seat Manager Authority: ",
+    seatManagerObject.authority.toBase58()
+  );
+}
+
+export async function claimSeat() {
+  let endpoint = "https://api.devnet.solana.com";
+  const connection = new Connection(endpoint);
+
+  const traderKeypair = new Keypair();
+  const phoenix = await Phoenix.Client.create(
+    connection,
+    endpoint,
+    traderKeypair.publicKey
+  );
+  console.log("Trader pubkey: ", traderKeypair.publicKey.toBase58());
+  let airdropSig = await phoenix.connection.requestAirdrop(
+    traderKeypair.publicKey,
+    1_000_000_000
+  );
+
+  let marketAddress = new PublicKey(
+    "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
+  );
+  let claimSeatIx = getClaimSeatIx(marketAddress, traderKeypair.publicKey);
+
+  let tx = new Transaction().add(claimSeatIx);
+
+  const txId = await phoenix.connection.sendTransaction(tx, [traderKeypair], {
+    skipPreflight: true,
+  });
+  await phoenix.connection.confirmTransaction(txId, "confirmed");
+  console.log("Claim Seat Tx: ", txId);
+
+  await phoenix.refreshMarket(marketAddress);
+  let marketData = phoenix.markets.get(marketAddress.toBase58())?.data;
+  if (!marketData)
+    throw new Error(
+      "Market data not found for market: " + marketAddress?.toBase58()
+    );
```

Comment:
will this ever fail? if the market doesn't exist, won't the tx fail in line 75?

Reply:
Yep this is unnecessary; removing